acquisitions/src/main/resources/thunderjet/consortia/features/prf-open-order-with-many-locations-from-different-tenants.feature

```diff
@@ -0,0 +1,314 @@
+Feature: Open order with many locations from different tenants
+
+  Background:
+    * url baseUrl
+    * call login consortiaAdmin
+    * configure headers = { 'Content-Type': 'application/json', 'Authtoken-Refresh-Cache': 'true', 'x-okapi-token': '#(okapitoken)', 'x-okapi-tenant': '#(centralTenant)', 'Accept': 'application/json' }
+    * configure retry = { count: 5, interval: 1000 }
+
+    * callonce variables
+    * callonce variablesCentral
+    * callonce variablesUniversity
+
+    * def fundId = callonce uuid1
+    * def budgetId = callonce uuid2
+
+    * def orderId = call uuid
```

Comment:
Damien would've recommended to move all UUID generation in each Scenario for uniqueness & also can add @Positve/@Negative indicators for each Scenario

Reply:
it make sense, added notation and moved uuid generation while keeping uuids that will be used in each scenario. Thanks